dmutils/status.py

```diff
@@ -39,7 +43,18 @@ def __init__(self, message):
 def _perform_additional_checks(additional_checks, response_data, error_messages):
     for additional_check in additional_checks:
```

Comment:
When do we actually carry out `additional_checks`? Is it just the APIs?

Reply:
Good question. I just went and checked - it is indeed just the APIs ATM.

Reply:
(search api checks es, api checks pg, av api checks clamd)